source/guide_coturn.rst

```diff
@@ -81,6 +82,7 @@ For some ciphers we need a DH-file:.
 
 Create a new config file at ``$HOME/etc/coturn/turnserver.conf``
 Replace values in brackets ``<value>`` with your values.
+Note: ``total-quota`` is the limit for all simultanious udp connections. In this example we only opened three ports for UDP, so this limit is 3. If you open more ports for UDP connections, change this setting accordingly.
```

Comment:
Please make this an actual `.. note::` box. You can copy the syntax from the "For this guide you should be familiar with (...)" block at the start of your guide.